src/App.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import App from './App';
+
+const WAIT_TIMEOUT_MS = 3000;
+
+const apiConfig = { BATCH_DUMP_TIMEOUT_MS: 1, BATCH_URL: '/file-batch-api' };
+jest.mock('./api-client/consts', () => apiConfig);
+
+describe('App component', () => {
```

Comment:
I should have mocked the endpoint like in the other test.
We want unit tests to be fast and reliable.

So, for a production use I would have either moved this test to `intergration-tests` directory or mocked the api responses.